static/src/javascripts/projects/common/modules/commercial/commercial-features.ts

```diff
@@ -82,6 +84,28 @@ class CommercialFeatures {
 			!isHosted &&
 			!newRecipeDesign;
 
+		if (isArticle && !this.articleBodyAdverts) {
+			const articleBodyAdvertsFalseIfTrue: Record<string, boolean> = {
+				forceAdFree,
+				isAdFreeUser: isAdFreeUser(),
+				sensitiveContent,
+				isMinuteArticle,
+				isLiveBlog,
+				isHosted,
+				newRecipeDesign: !!newRecipeDesign,
+			};
+
+			const articleBodyAdvertsFalseIfFalse: Record<string, boolean> = {
+				'switches.commercial': switches.commercial,
+				externalAdvertising: externalAdvertising,
+			};
+
+			log('commercial', 'articleBodyAdverts = false because:', {
+				...pickBy(articleBodyAdvertsFalseIfTrue, (v) => v),
```

Comment:
I wonder if we could make these messages less tied to the implementation details e.g. "Adverts not shown because" rather than "articleBodyAdverts = false because".
We might know about this variable but it would be nice if a consumer of the log didn't have to!